modules/lib/lib-admin/README.md

```diff
@@ -48,13 +47,11 @@ objects, e.g. `log`, `app`, `__`, etc, requiring library by the default path wil
 const {getBaseUri, getAssetsUri, getHomeToolUrl, HomeToolUrlType} = require('/lib/xp/admin');
 ```
 
-More detailed explanation on how it works and how to type custom import function can be
-found [here](https://github.com/enonic/xp/tree/master/modules/lib/typescript/README.md).
+More detailed explanation on how it works and how to type custom import function can be found [here](https://github.com/enonic/xp/tree/master/modules/lib/README.md).
```

Comment:
I guess, it is better to refer to our official docs instead of github

Reply:
Replaced with a link to API. Note, that [TS chapter](https://github.com/enonic/doc-xp/pull/431) is not yet merged and deployed.